docs/manual/common/concepts/ES_CQRS.md

```diff
@@ -1,6 +1,6 @@
 # Managing data persistence
 
-When designing your microservices, remember that each service should own its data and have direct access to the database. Other services should then use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too-tight coupling between the services. In this way, each microservice defines a bounded context, similar to [Bounded Contexts](http://martinfowler.com/bliki/BoundedContext.html) in Domain-Driven Design.
+When designing your microservices, remember that each service should own its data and have direct access to the database. Other services should then use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too-tight coupling between the services. In this way, each microservice operates within a clear boundary, which is known as a [Bounded Context](http://martinfowler.com/bliki/BoundedContext.html) in Domain-driven Design.
```

Comment:
I preferred the vagueness of the original wording. Originally we stated a uService is bounded in a similar way than DDD's BC, but the new text seems to imply they are exactly the same thing.